mautic/mautic
# Add environment variables for running test suite.

Add base environment variables file.
Update setup file to copy it at appropriate place.

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | "features" for all features, enhancements and bug fixes (until 3.3.0 is released) 
| Bug fix?                               | yes
| New feature?                           | no
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | no (Not required)
| Related user documentation PR URL      | N/A
| Related developer documentation PR URL | N/A
| Issue(s) addressed                     | Fixes #... 

#### Description:
Adding sample environment variables that can be picked up while running PHPUnit test cases on local.

#### Steps to test this PR:
1. Pull the PR on local. We can't test it on MautiBox as this is related to local setup.
2. Run ` sh .ddev/mautic-setup.sh`
3. When you run above script, ensure that ddev setup is not pre-configured. 
4. This will copy `.env.dist` from `.ddev` folder to the main folder of the project.
